### ISSUE TITLE
Node Sent E-mail  (error)

### ISSUE THREAD

**Issue by magadan** [n8n-io/n8n#17]
Results: 1  
ERROR: 139622690989888:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:
Error: 139622690989888:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:

maybe my nodejs version ?

**Comment by janober** [comment]
Not sure. Normally you should try to run it in Node 10 as that is the current version and the one n8n was developed with.

**Comment by magadan** [comment]
now im using v12.9.1

**Comment by magadan** [comment]
i will reinstall in a new enviroment with node 10.

**Comment by janober** [comment]
Ah yes you are right. Did not realize that 12 is the now current version. But never tried with that one either. Would really advice to run it in Node 10 for now as it will work in that version for sure.
Like written in the other issue I would strongly advice everybody to run n8n in Docker. It is the simplest way to get n8n up and running and will then run in a correct environment for sure.

**Comment by magadan** [comment]
ok, i will install docker-ce and deploy with portainer. thanks.

**Comment by RicardoE105** [comment]
Closing this issue.

**Comment by andyvanee** [comment]
I came across the same error message using the latest Docker image https://hub.docker.com/r/n8nio/n8n which is using node 12. 

```
docker run --rm n8nio/n8n node -v
v12.19.0
```

For a bit more context, I was trying to send via Office365, which uses the following SMTP settings.

```
Server name: smtp.office365.com
Port: 587
Encryption method: STARTTLS
```

After a bit of reading I found that STARTTLS actually begins insecure and then upgrades the connection, so I turned SSL/TLS off in my n8n credentials and it now works!
